lib/Sema/CSSolver.cpp

```diff
@@ -2320,17 +2155,6 @@ Constraint *ConstraintSystem::selectDisjunction() {
   if (disjunctions.empty())
     return nullptr;
 
-  // Attempt apply disjunctions first. When we have operators with
-  // designated types, this is important, because it allows us to
-  // select all the preferred operator overloads prior to other
-  // disjunctions that we may not be able to short-circuit, allowing
-  // us to eliminate behavior that is exponential in the number of
-  // operators in the expression.
-  if (getASTContext().TypeCheckerOpts.SolverEnableOperatorDesignatedTypes) {
-    if (auto *disjunction = selectApplyDisjunction())
```

Comment:
I think `selectApplyDisjunction` could also be removed now right?

Reply:
Ah you're right, thanks for catching that!